tools/clickbench/send_datawithidx.py

```diff
@@ -0,0 +1,66 @@
+import json
+import requests
+import os
+import time
+
+# Path to hits.json file
+ndjson_file = "hits.json"
+
+# Target URL for HTTP POST request
+url = "http://localhost:8081/elastic/_bulk"
+
+# index string
+index_string = '{ "index" : { "_index" : "hits" } }\n'
+
+buffer_limit = 10 * 1024 * 1024
+
+def post_data(buffer):
+    """Function to send HTTP POST request with buffer as payload."""
+    try:
+        response = requests.post(url, data=buffer)
+        if response.status_code != 200:
+            print(f"Failed to post data. Status code: {response.status_code}")
+    except requests.exceptions.RequestException as e:
+        print(f"Request failed: {e}")
+
+def process_ndjson(file_path):
```

Comment:
nit: this looks quite similar to the `process_ndjson()` in fix_hits.py. Maybe this duplication should be removed?